nodes/arm64_ubuntu_16_04_LTS_android.json

```diff
@@ -0,0 +1,18 @@
+{
+  "contact": {
+    "name": "Andrew Breckenridge",
+    "email": "swiftci@fb.com",
+    "company": "Facebook, Inc."
+  },
+  "node": {
+      "platform": "Android arm64",
+      "os_version": "Ubuntu 16.04 LTS"
+  },
+  "jobs": [
+    {
+      "display_name": "Swift - Android (Tools RA, Stdlib RD) (master)",
+      "branch": "master",
+      "preset": "buildbot_linux_crosscompile_android,tools=RA,stdlib=RD,build"
```

Comment:
We probably want to change the display name and the preset. The Android ARM bot also has the same problem.

Even if the name and the preset name say tools are  Release+Assertions and stdlib is Release+DebugInfo, it seems to me that the preset is setting everything Release+Assertions.

The preset will have to change to the preset that compiles ARM64.